### DISCUSSION TITLE
Trumpler, K-effect, reddening

### DISCUSSION THREAD

**Discussion by sahil5d** [a-cosmology-group/acg#271] [Redshift Models]
I came across an interesting history of the K-effect or Trumpler effect, and the history of reddening (also studied by Robert J. Trumpler).

### 1) K-effect

_"The K term originally named by Campbell in 1911, is an excess redshift always seen in the spectra of high luminosity (O and B) stars. It amounts to about 10km sec^-1 or z_u = 0.00003. While it is very small, the value is well determined, and it is highly significant at the 10σ level (cf Trümpler 1956)."_
Burbidge, 1994 
https://adsabs.harvard.edu/full/1996IAUS..168..407B

And Arp's 1992 paper on the effect, 
https://adsabs.harvard.edu/full/1992MNRAS.258..800A.

(By the way, I haven't found any information on Trumpler's views on tired light.)

### 2) Reddening

<img width="607" alt="trumpler distances" src="https://github.com/user-attachments/assets/36d62e11-fb33-47f2-8c93-01eb19b8d355">

_"Unless we are willing to admit that the dimensions of open clusters depend on their distance from the Sun, we are led to the conclusion that the inverse square law on which the photometric distances are based does not hold and that a general absorption is taking place within our stellar system."_
Trumpler, 1930 
https://iopscience.iop.org/article/10.1086/124039 
(found via http://spiff.rit.edu/classes/phys230/lectures/ism_dust/ism_dust.html)

This kind of reasoning is beautifully analogous to tired light. That instead of a physical absurdity, a new effect proportional to the distance traveled can explain the data. It's astonishing that standard model astrophysicists who study reddening and should know its history haven't applied the same thinking to cosmic redshift.

**Comment by LifeIsStrange** [comment]
There are almost no papers on the K-effect/trumpler effect since decades, anomalous redshifts are among the most ignored major issues in cosmology, and is related but distinct of quantized redshift.
Most recent papers I could find:
https://arxiv.org/abs/1306.1015
This paper shows that anomalous redshift increase with temperature.
Since temperature is rhoughly synonymous with energy and since one possible family of tired light mechanism is to have a variable speed of light, the observation of the K-effect can be connected with the empirical evidence for a variable speed of light.
There are multiple theories of variable speed of light, some says that the light speed is dependent on the source (emitting material) speed, which is called the emitter theory.
Nobody remember the emitter theory nowadays yet it was a leading theory and one of the only one, along with relativity, to pass the Michelson–Morley experiment. (btw if someones could send me the pdf of this paper: https://link.springer.com/article/10.1140/epjp/s13360-024-05529-w that would be very kind)
Anyway, the emitter theory has been ruled out but NOT the stochastic emitter theory
https://www.scilit.net/publications/034c9268b99696a4d048c64b777e2b80
The other way to variate the speed of light is to say it is energy dependent, and indeed corroborate the k-effect.
What is very interesting, is that there seems to be "robust evidence" for the speed of light being energy dependent:
https://arxiv.org/abs/1607.08043

other papers about anomalous redshift
https://www.semanticscholar.org/paper/A-Review-of-Anomalous-Redshift-Data-Ratcliffe/2f1ecc91276f5b17822265f30999619acf28b9c1
https://arxiv.org/abs/astro-ph/0509630
https://www.semanticscholar.org/paper/Study-of-Possible-Local-Quasars-IV.-Effects-of-in-Panov/ce2732991343430e96a2e1da1acb91d8739551ec
https://www.researchgate.net/publication/323780835_Compelling_Evidence_of_a_Static_Universe_by_the_Karlsson_Peaks_in_the_Quasar's_Redshift_Distribution

And of course the incredible work of Shamir
https://arxiv.org/search/astro-ph?searchtype=author&query=Shamir,+L
and https://www.medjchem.com/index.php/Medjphysics/article/view/518
https://arxiv.org/abs/2203.02282
